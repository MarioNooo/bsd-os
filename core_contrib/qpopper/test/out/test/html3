Return-Path: Bahai-Tech-x@Bounty.BCCA.Org
Received: from oneworld.wa.com (oneworld.new-era.com [208.133.44.6]) by new-era.com (8.7.6/8.7.3) with SMTP id XAA01020 for <lgl@island-resort.com>; Tue, 1 Apr 1997 23:23:38 -0500
Received: by oneworld.wa.com (Smail3.1.29.1/NeXT 3.2)
	id m0wCHTG-00074HC; Tue, 1 Apr 97 20:16 PST
X-Loop: Bahai-Tech-request@Bounty.BCCA.Org
Errors-To: Bahai-Tech-x@Bounty.BCCA.Org
Sender: Bahai-Tech-x@Bounty.BCCA.Org
X-IMPORTANT-NOTE: Please send unsubscribe messages
		  to Bahai-Tech-Request@BCCA.Org
To: "Baha'i Tech" <Bahai-Tech@BCCA.Org>
Content-Transfer-Encoding: 7bit
X-Msmail-Priority: Normal
Mime-Version: 1.0
X-Priority: 3
Message-Id: <199704020423.XAA05601@mail.travel-net.com>
Subject: Re: US map needed
X-Mailer: Microsoft Internet Mail 4.70.1161
Date: Tue, 1 Apr 1997 23:15:00 -0500
From: "Jim Desson" <jdesson@redoak.ca>
Content-Type: text/html; charset=ISO-8859-1
Content-Transfer-Encoding: quoted-printable

<html><head></head><BODY bgcolor=3D"#FFFFFF"><p><font size=3D2 =
color=3D"#000000" face=3D"Arial">This site( =
http://www.graphicmaps.com/webimage/us.html ) may be helpful. I didn't =
notice any city references though.<br><br>----------<br>&gt; From: Marc =
Lustig &lt;<font =
color=3D"#0000FF"><u>mlustig@unidui.uni-duisburg.de</u><font =
color=3D"#000000">&gt;<br>&gt; To: Baha'i Tech &lt;<font =
color=3D"#0000FF"><u>Bahai-Tech@BCCA.Org</u><font =
color=3D"#000000">&gt;<br>&gt; Subject: US map needed<br>&gt; Date: =
April 1, 1997 8:28 AM<br>&gt; <br>&gt; Dear Friends,<br>&gt; <br>&gt; =
for the purpose of Baha'i-Announce I am looking for a political map of =
the <br>&gt; US. I need only the state-borders and major cities.<br>&gt; =
<br>&gt; Any idea where to find this on the Internet ?<br>&gt; <br>&gt; =
Thanks !<br>&gt; &nbsp;&nbsp;Marc<br>&gt; <br>&gt; =
_________________________________________________________<br>&gt; =
&nbsp;&nbsp;Marc Lustig - Student of Social Sciences and =
Philosophy<br>&gt; <br>&gt; &nbsp;&nbsp;Visit Episteme-X !<br>&gt; =
&nbsp;&nbsp;<font =
color=3D"#0000FF"><u>http://www.episteme-x.net</u><font =
color=3D"#000000"> <br>&gt; =
_________________________________________________________</p>
</font></font></font></font></font></font></font></body></html>

