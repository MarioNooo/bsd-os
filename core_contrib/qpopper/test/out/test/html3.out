To: "Baha'i Tech" <Bahai-Tech@BCCA.Org>
Subject: Re: US map needed
Date: Tue, 1 Apr 1997 23:15:00 -0500
From: "Jim Desson" <jdesson@redoak.ca>

This site( http://www.graphicmaps.com/webimage/us.html ) may be helpful. I didn't notice any city references though.

----------
> From: Marc Lustig <mlustig@unidui.uni-duisburg.de>
> To: Baha'i Tech <Bahai-Tech@BCCA.Org>
> Subject: US map needed
> Date: April 1, 1997 8:28 AM
> 
> Dear Friends,
> 
> for the purpose of Baha'i-Announce I am looking for a political map of the 
> US. I need only the state-borders and major cities.
> 
> Any idea where to find this on the Internet ?
> 
> Thanks !
>   Marc
> 
> _________________________________________________________
>   Marc Lustig - Student of Social Sciences and Philosophy
> 
>   Visit Episteme-X !
>   http://www.episteme-x.net 
> _________________________________________________________

